src/main/kotlin/no/nav/syfo/oversikthendelsetilfelle/PersonOppfolgingstilfelleHendelseQueries.kt

```diff
@@ -181,17 +187,19 @@ fun DatabaseInterface.oppdaterPersonOppfolgingstilfelleNyEnhetMottatt(personId:
 
 const val queryOppdaterPersonOppfolgingstilfelleMottatt = """
                         UPDATE PERSON_OVERSIKT_STATUS
-                        SET tildelt_enhet = ?, sist_endret = ?
+                        SET tildelt_enhet = ?, tildelt_enhet_updated_at = ?, sist_endret = ?
                         WHERE fnr = ?
                 """
 
 fun DatabaseInterface.oppdaterPersonOppfolgingstilfelleMottatt(personId: Int, oversikthendelsetilfelle: KOversikthendelsetilfelle) {
     val tidspunkt = Timestamp.from(Instant.now())
+    val now = Timestamp.from(OffsetDateTime.now().toInstant())
     connection.use { connection ->
         connection.prepareStatement(queryOppdaterPersonOppfolgingstilfelleMottatt).use {
             it.setString(1, oversikthendelsetilfelle.enhetId)
-            it.setTimestamp(2, tidspunkt)
-            it.setString(3, oversikthendelsetilfelle.fnr)
+            it.setTimestamp(2, now)
```

Comment:
Her også: Bruke setObject med en instans av OffsetDateTime - samme på linje 178

Reply:
Ref kommentarer om get/setObject istedet for Timestamp:
Ser ut som om det vil fungerer som tenkt med setObject(altså at jeg får UTC lagret), siden jeg nå setter UTC eksplisitt, men ser ut som om man da potensielt vil få feil tiddsone(ikke UTC, e.g. FInland pga GCP-loksjon) forrdi man da ikke eksplisitt setter offset til UTC.

Ref: https://www.postgresqltutorial.com/postgresql-timestamp/#:~:text=PostgreSQL%20provides%20you%20with%20two%20temporal%20data%20types,it%20does%20not%20have%20any%20time%20zone%20data.

Enig eller uenig i det resonnementet?

Reply:
Ikke helt enig: Vi styrer dette selv om tidssone satt i docker-konfig'en: Så lenge vi bruker
FROM navikt/java:17
så vil default tidssone være Europe/Oslo - se linje 18 her:
https://github.com/navikt/baseimages/blob/master/java/Dockerfile

Hvis vi begynner å bruke et annet base-image må vi vurdere dette og ta stilling til om vi skal fortsette med Europe/Oslo eller går over til UTC.

Reply:
Etter diskusjon: Går her for get/setObject som Geir foreslår i

Reply:
Endre til getObject og setObject med OffsetDateTime nå.